waku/node/v2/rpc/wakurpc.nim

```diff
@@ -27,11 +27,20 @@ proc setupWakuRPC*(node: WakuNode, rpcsrv: RpcServer) =
     let wakuRelay = cast[WakuRelay](node.switch.pubSub.get())
     # XXX also future return type
     # TODO: Shouldn't we really be doing WakuNode publish here?
+    debug "waku_publish", topic=topic, payload=payload
     discard wakuRelay.publish(topic, payload)
     return true
     #if not result:
     #  raise newException(ValueError, "Message could not be posted")
 
+  rpcsrv.rpc("waku_publish2") do(topic: string, message: WakuMessage) -> bool:
+    # XXX also future return type
+    debug "waku_publish2", topic=topic, message=message
```

Comment:
wakunode2 import missing, but can't add it as `wakurpc` is imported already in `wakunode2`